index.js

```diff
@@ -14,19 +14,34 @@ const moons = [
 
   You should not need to make any other changes to the code.
 */
-function organizeMoonsByPlanet(moons) {
+//function organizeMoonsByPlanet(moons) {
+  //const result = {};
+  //for (let moon of moons) {
+    //const name = moon.name;
+   // const planet = moon.planet;
+   // if (result[planet]) {
+     // result[planet].push(name);
+   // } else {
+    //  result[planet] = [name];
+    //}
+ // }
+  //return result;
+
+
+
+const organizeMoonsByPlanet = (moons) => {
+ const {name, planet} = moons
   const result = {};
   for (let moon of moons) {
-    const name = moon.name;
-    const planet = moon.planet;
+    const {name,planet} = moon
```

Comment:
Great job on this!

How many `name` and `planet` variables do we need? Which pair hold which values? If any are unneeded, remove them!